geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/hash/AbstractHashesIntegrationTest.java

```diff
@@ -111,6 +111,22 @@ public void testHMSet() {
     assertThat(jedis.hlen(key)).isEqualTo(hash.size());
   }
 
+  @Test
+  public void testHMSetErrorMessage_givenIncorrectDataType() {
+    Map<String, String> animalMap = new HashMap<>();
+    animalMap.put("chicken", "eggs");
+
+    jedis.set("farm", "chicken");
+    assertThatThrownBy(() -> jedis.hmset("farm", animalMap))
+        .isInstanceOf(JedisDataException.class)
+        .hasMessageContaining("WRONGTYPE Operation against a key holding the wrong kind of value");
+
+    jedis.sadd("zoo", "elephant");
+    assertThatThrownBy(() -> jedis.hmset("zoo", animalMap))
+        .isInstanceOf(JedisDataException.class)
+        .hasMessageContaining("WRONGTYPE Operation against a key holding the wrong kind of value");
```

Comment:
Just realized you could actually change this to `.hasMessage` instead of `.hasMessageContaining` in case anything ever gets appended to the beginning or end.  We could also use the constant from `RedisConstants` here:
```
.hasMessage("WRONGTYPE " + ERROR_WRONG_TYPE)
```

so if the message ever changed for some reason it would be less work to update it throughout all the tests.

Reply:
I think I just did this in the HVALS PR